students/piotrowski_stanislaw/lesson_01_basics/L1_IFN_06.py

```diff
@@ -0,0 +1,15 @@
+#https://snakify.org/lessons/integer_float_numbers/problems/car_route/
+#piotrsta
+
+from math import ceil
```

Comment:
to jest poprawny sposób na import, ale czasami lepiej użyć: `import math` i wywołania `math.ceil(liczba)`, żeby mieć pewność, że używana funkcja pochodzi z namespace/modułu, który chcemy użyć.

Reply:
OK. Będę się stosował.